src/components/popup/intro/intro.jsx

```diff
@@ -27,7 +29,7 @@ export default class Intro extends Component {
 		return (
 			<div class={style.intro}>
 				<div class={style.title}>
-					<LocalLabel localizeKey='title'>Thanks for visiting</LocalLabel> {DOMAIN}
+					<LocalLabel localizeKey='title'>Thanks for visiting</LocalLabel> {locale !== 'fr' && (DOMAIN)}
```

Comment:
Will this still print `DOMAIN` if `locale` does not equal `fr`?

Reply:
Yes it does, it has been tested:
https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator